### ISSUE TITLE
[Screen Reader – Make Code – Extensions] Screen reader does not provide placeholder information at search field under 'Extensions' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3696]
**User Experience:**
If the correct  information and relationships are not provided, it would be difficult for people with different disabilities as the information and relationships that are implied by visual or auditory formatting will not be preserved when the presentation format changes

**Note**: User credentials should NOT be included in the bug.

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Verify that screen reader announces place holder information at search field or not.

**Actual Result:**
Screen reader does not provide placeholder information at search field under 'Extensions' screen.

**Expected Result:**
Screen reader should provide placeholder information at search field under 'Extensions' screen. It should be read as 'Search or enter project url' edit field.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

![WCAG1 3 1_Extensions_Search field](https://user-images.githubusercontent.com/72871089/99041153-69892700-25b0-11eb-8301-0972c53d11da.jpg)
